pyproject.toml

```diff
@@ -53,13 +48,6 @@ taskipy = "^1.2.0"
 pytest-black = "^0.3.11"
 black = { version = "*", allow-prereleases = true }
 pytest-isort = "^1.0.0"
-sphinx = "^3.2.1"
```

Comment:
looks like you're removing the docs libraries - are there instructions somewhere how to install these if you want to build the docs?

Reply:
The docs requirements were moved into the `docs` package and should have been removed from here: https://github.com/Bellman-devs/bellman/blob/develop/docs/docs_requirements.txt

This PR is catching up with that task.

Reply:
we need to leave these in here, we use poetry to build docs as well and buidling docs in general I would consider develoment dependencies, so they should stay